src/app/legacy/containers/Headings/__snapshots__/index.test.jsx.snap

```diff
@@ -589,6 +589,7 @@ exports[`Headings with plain text subheadline should render h2 containing correc
 
 @media (min-width: 37.5rem) {
   .emotion-2 {
+    padding-top: 0.5rem;
```

Comment:
We identified a bug in the code - see comment above (fix here: https://github.com/bbc/simorgh/blob/b3f64fd5eb6876b43cd4c50c5190fd15cac761ab/src/app/legacy/psammead/psammead-headings/src/index.jsx#L39-L40